ngraph_creator/operations/src/AveragePool2D.cpp

```diff
@@ -120,8 +120,11 @@ std::shared_ptr<ngraph::Node> AveragePool2D::createNode() {
         }
     }
 
-    if (!useNchw) {  // No conversion needed if useNchw set
-        inputNode = transpose(NHWC_NCHW, inputNode);
+    if (!transposed_nchw) {
+        if (!useNchw) {  // No conversion needed if useNchw set
+            inputNode = transpose(NHWC_NCHW, inputNode);
+            transposed_nchw = true;
+        }
```

Comment:
Can this code be shared by multiple operations? It seems like its needed in all operations, maybe we can have a function in the parent class?